docs/querying-data.rst

```diff
@@ -0,0 +1,196 @@
+
+.. tip::
+
+  The ``tender``, ``award`` and ``contract`` objects in OCDS all have a ``.status`` property.
+
+  Kingfisher Views exposes these status properties in ``tender_summary.tender_status``, ``awards_summary.award_status`` and ``contracts_summary.contract_status``.
+
+  Consider which statuses you want to include or exclude from your analysis, for example you might wish to exclude pending and cancelled contracts when calculating the total value of contracts for each buyer.
+
+  The `OCDS codelist documentation <https://standard.open-contracting.org/latest/en/schema/codelists/#>`__ describes the meaning of the statuses for each object.
+
+Calculate the top 10 buyers by award value
+------------------------------------------
+
+Summary data on buyers is stored in the ``buyer_summary`` table and summary data on awards is stored in the ``award_summary`` table.
+
+To join summary tables, use the ``id`` column, which uniquely identifies a release. To learn more about the relationships between tables refer to the :ref:`relationships` documentation.
+
+The ``buyer_summary`` table doesn't include the buyer's name, however the ``buyer`` column contains a jsonb blob of the buyer  for each contracting process, which this can be extracted from.
+
+Most summary tables in Kingfisher Views include a column containing jsonb blobs of the object the summary relates to.
+
+The following query calculates the top 10 buyers by award value for collection ``1259``, disaggregated by currency and counting 'active' awards only:
+
+.. code-block:: sql
+
+  select
+      buyer_identifier,
+      buyer -> 'name' as buyer_name, --extract the buyer name from the JSON
+      award_value_currency,
+      sum(award_value_amount) as award_amount
+  from
+      awards_summary
+  join
+      buyer_summary on awards_summary.id = buyer_summary.id
+  where
+      awards_summary.collection_id = 1259
+  and
+      awards_summary.award_value_amount > 0 --filter out awards with no value
+  and
+      awards_summary.award_status = 'active'
+  group by
+      buyer_identifier,
+      buyer_name,
+      award_value_currency
+  order by
+      award_amount desc
+  limit
+      10;
+
+Check which fields are available
+--------------------------------
+
+Use the `OCDS schema documentation <https://standard.open-contracting.org/latest/en/schema/release/>`__ to understand the meaning, structure and format of the fields in OCDS and to identify the fields needed for your analysis.
+
+Coverage of the OCDS schema varies by publisher. Use the ``field_counts`` table to check whether the fields needed for your analysis are available.
+
+The following query lists the coverage of each field in the current schema:
+
+.. code-block:: sql
+
+  select
+    *
+  from
+    field_counts
+
+For schemas with multiple collections, use the ``collection_id`` column to filter your results for a particular collection.
```

Comment:
This is good! I sometimes forget to filter by `collection_id` in these schemas.

Point for future review though; we should be very clear about what "a schema" is in this context. I usually use the terminology "a view" when referring to a kf-views schema. I think that would make the line "For views with multiple collections…" read a bit more naturally in context. However it's not particularly unclear atm.

Reply:
That said we've done some setup work saying "KF views creates database schemas…" on line 18 so I think this is fine. Just a broader point for thinking about in the future.

Reply:
It's also covered in https://kingfisher-views.readthedocs.io/en/latest/#what-s-a-view which is linked in 'before you begin' at the top of the page